tjreports/site/models/reports.php

```diff
@@ -55,6 +55,16 @@ class TjreportsModelReports extends JModelList
 	// Used for to limit query
 	protected $canLimitQuery = false;
 
+	public $customFieldsTable = '';
+
+	public $customFieldsTableAlias = '';
+
+	public $customFieldsTableExists = false;
```

Comment:
Didn't we decide to remove this variable ? Or make it private/protected so it cannot be overwritten by child models.

Reply:
changed